constraints.md

```diff
@@ -252,18 +246,19 @@ Because the Foreign Key constraint requires per-row checks on two tables, statem
 
 We're currently working to improve the performance of these statements, though.
 
+{{site.data.alerts.callout_success}}You can improve the performance of some statements that use Foreign Keys by also using <code><a href="interleave-in-parent.html">INTERLEAVE IN PARENT</a></code>.{{site.data.alerts.end}}
```

Comment:
Most of the other links are anchored as "Interleaved Tables". Is this one intentionally different?